iothub/device/src/DeviceStreamResponse.cs

```diff
@@ -1,37 +1,30 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// The data structure represent the stream response that is used for interacting with IotHub.
+    /// The data structure to represent the stream response that is used for interacting with Iot Hub.
     /// </summary>
-    public sealed class DeviceStreamResponse
+    internal sealed class DeviceStreamResponse
```

Comment:
Is this class of any value to the user?

Reply:
It isn't exposed in our convenience layer, so I figured no.